G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.settitle/ShouldGetTitle_WhenSetTitle.G1ANT

```diff
@@ -0,0 +1,13 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
+♥valueToBePlaced = ‴G1ANT‴
+
+googlesheet.open ♥FileID
+googlesheet.gettitle result ♥before
+googlesheet.settitle ♥valueToBePlaced
+googlesheet.gettitle ♥res
+googlesheet.close
+test condition ⊂♥before == ♥res⊃
```

Comment:
That should fail - the title should be updated, so before != after, but you re-set the same title... BTW how do you know that the "before" title wasn't set to G1ANT before the test started? IMHO the test should be more careful:
- set the title to A
- get the title
- set the title to B
- get the title
- compare the titles you loaded

or
- get the title
- set it to title + random
- get the title
- compare

or easiest:
- set the title to long random string (like guid)
- read the title
- verify it was set to your guid